Rogue-Ape-Studios/Crusader
# ✨ Attack direction

## Content

Adds the function for the player body to rotate, following the mouse, to look in the direction you are dashing and attacking.

## Changes

### Added
`Assets/Scripts/Camera/CameraFollow.cs` : Was created. Camera now follows the player without rotating.
`Assets/Prefabs/Camera/PlayerCamera.prefab` : Was created. For simplicity, it is a camera with 1 added script to follow the player.

### Updated
`Assets/Prefabs/Player/BasicAttack.anim` : Was updated. Forward movement has been removed from the animation.
`Assets/Prefabs/Player/BasicAttack_1.anim` : Was updated. Forward movement has been removed from the animation.
`Assets/Prefabs/Player/BasicAttack_2.anim` : Was updated. Forward movement has been removed from the animation.
`Assets/Scripts/Player/Combat/BasicAttack.cs` : Was updated. Now has a way to make the player move forward during attacks.
`Assets/Scripts/Player/Movement/PlayerController.cs` : Was updated. Now has a way to rotate the player's body and can stop reading inputs during attack animations.


## Testing

The feature can be testing by following these steps:

1. Place the updated player prefab in the scene.
2. Check the inspector for references and values.
3. Play.

Closes #24 #53
